src/driver/ydb/YdbQueryRunner.ts

```diff
@@ -60,15 +83,52 @@ export class YdbQueryRunner extends BaseQueryRunner implements QueryRunner {
         }
     }
 
-    connect(): Promise<any> {
-        throw new Error("Method not implemented.")
+    /**
+     * Creates/uses database connection from the connection pool to perform further operations.
+     * Returns obtained database connection.
+     */
+    async connect(): Promise<Ydb.Session> {
+        if (!this.driver.driver) {
+            throw new DriverNotInitialized("ydb")
+        }
+        if (this.databaseConnection && !this.isReleased)
+            return this.databaseConnection
+
+        if (!this.databaseConnection) {
+            this.databaseConnection =
+                await this.driver.driver.tableClient.getSessionUnmanaged(
+                    this.sessionTimeout,
+                )
+        }
+
+        this.databaseConnection.acquire()
+
+        this.isReleased = false
+
+        this.driver.connectedQueryRunners.push(this)
+
+        return this.databaseConnection
     }
 
+    /**
+     * Releases used database connection.
+     * You cannot use query runner methods once its released.
+     */
     async release(): Promise<void> {
-        await this.driver.disconnect()
+        if (this.isReleased || !this.databaseConnection) {
```

Comment:
Please use here QueryRunnerAlreadyReleasedError

Reply:
No, it`s no error when we are releasing QueryRunner